dev/ci/ci-fiat_crypto_ocaml.sh

```diff
@@ -17,5 +17,6 @@ ulimit -s 131072
 make_args=(EXTERNAL_REWRITER=1 EXTERNAL_COQPRIME=1 OCAMLFIND=ocamlfind)
 
 ( cd "${CI_BUILD_DIR}/fiat_crypto"
-  make "${make_args[@]}" -j 1 standalone-ocaml lite-generated-files
+  make "${make_args[@]}" -j 1 standalone-ocaml
```

Comment:
```suggestion
```
Let's try skipping this step, and only compiling the particular binaries we need.  This should shave most of the time off.